pages/lists/[listId]/edit.tsx

```diff
@@ -207,133 +215,194 @@ export default function EditListPage({ list, initialContributors }: EditListPage
     );
   };
 
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
+  const [deleteLoading, setDeleteLoading] = useState(false);
+
+  const handleOnDelete = () => {
+    setIsDeleteModalOpen(true);
+  };
+
+  const handleDeleteModalClose = () => {
+    setIsDeleteModalOpen(false);
+  };
+
+  const handleOnDeleteConfirm = async () => {
+    setSubmitted(true);
+    setDeleteLoading(true);
+
+    const { error } = await fetchApiData({
```

Comment:
There's a couple build errors at the moment.

```
./pages/lists/[listId]/edit.tsx:239:9
Type error: Type 'string | null | undefined' is not assignable to type 'string'.
  Type 'undefined' is not assignable to type 'string'.

  237 |         path: "/lists/${list.id}",
  238 |         method: "DELETE",
> 239 |         bearerToken: sessionToken,
      |         ^
  240 |         pathValidator: () => true,
  241 |       });
  242 | 
```
and

```
./pages/lists/[listId]/edit.tsx
248:4  Error: Parsing error: 'catch' or 'finally' expected.
```

Reply:
I resolved those in my last commit can you please check once?